dolphinscheduler-master/src/test/resources/application.yaml

```diff
@@ -127,9 +127,7 @@ master:
   kill-application-when-task-failover: true
   registry-disconnect-strategy:
     # The disconnect strategy: stop, waiting
-    strategy: waiting
-    # The max waiting time to reconnect to registry if you set the strategy to waiting
-    max-waiting-time: 100s
+    strategy: stop
```

Comment:
Should we change alert-server and api-server?

Reply:
Only master/worker have this config.

Reply:
Okay.